README.md

```diff
@@ -119,7 +119,7 @@ The settings in the `shipit.yml` file relate to the different things you can do
 * [Installing dependencies](#installing-dependencies) (`dependencies`)
 * [Deployment](#deployment) (`deploy`, `rollback`, `fetch`)
 * [Environment](#environment) (`machine.environment`)
-* [CI](#ci) (`ci.hide`, `ci.allow_failures`)
+* [CI](#ci) (`ci.require`, `ci.hide`, `ci.allow_failures`)
```

Comment:
Writing the doc made me realize the previous ones were in the imperative form, so I renamed `ci.required` in `ci.require` for consistency